pages/blog/blogPage.html

```diff
@@ -0,0 +1,306 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" href="/css/base.css" />
+        <link rel="stylesheet" href="/css/container.css" />
+        <link rel="stylesheet" href="/css/header.css" />
+        <link rel="stylesheet" href="/styles.css" />
+        <link rel="stylesheet" href="/css/footer.css" />
+        <link rel="stylesheet" href="/css/responsive.css" />
```

Comment:
mấy component dùng chung mà page nào cũng có như container, header, footer thì @import vào trong base.scss, cũng ko cần biên dịch sang css rồi link như này
```suggestion
        <link rel="stylesheet" href="/css/base.css" />
        <link rel="stylesheet" href="/styles.css" />
```